database/index.js

```diff
@@ -14,12 +14,15 @@ const ReviewSchema = mongoose.Schema({
 
 const photoSchema = mongoose.Schema({
   place_id: {
-    type: String,
+    type: Number,
     unique: true,
+    index: true
   },
   place_name: String,
-  photos: [PhotosSchema],
-  reviews: [ReviewSchema],
+  photos: Array,
+  reviews: Array,
+ // photos: [PhotosSchema],
+ // reviews: [ReviewSchema],
```

Comment:
Do you still need these commented-out variables? If not, I would remove them - if you're looking to continue turning them on and off for further testing of functionality, I would put that in a new branch, rather than merging that to your master as a final product.